lisa/tools/ethtool.py

```diff
@@ -237,6 +246,7 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._device_features_map: Dict[str, DeviceFeatures] = {}
         self._device_link_settings_map: Dict[str, DeviceLinkSettings] = {}
         self._device_ring_buffer_settings_map: Dict[str, DeviceRingBufferSettings] = {}
+        self._device_gro_lro_settings_map: Dict[str, DeviceGroLroSettings] = {}
```

Comment:
How about to use one dict? So all information of a device can be put in the same structure. And the values can be optional, if it's not queried.

Reply:
ok, can add it in another commit on top of the gro_lro test case

Reply:
Do you mean in the same PR or next PR?

Reply:
next PR with more test cases added. But the base commit will be having one Dict for DeviceSettings